src/lib/utils/index.ts

```diff
@@ -8,25 +6,44 @@ interface DomainMetadataParams {
 	story: string;
 }
 
+interface UploadResponseInterface {
```

Comment:
Redundant name `UploadResponseInterface` is an interface.
Consider `UploadResponse` or `UploadResponseDto` (Dto = Data Transfer Object)